### PR TITLE
Mat 4294 remove tgt storage

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-4294](https://jira.cms.gov/browse/MAT-4294)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [x] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by RohitKandimalla** [comment]
Idk if this has to be covered in this PR, but I just want to point it out. @adongare Do we have a call to terminology-service using TGT in madie-patient right? which needs to be updated since TGT is no longer available in local storage.

**Review by RohitKandimalla — APPROVED** [review]

File: src/components/measureEditor/MeasureEditor.tsx [hunk 1]
```diff
@@ -105,16 +105,15 @@ const MeasureEditor = () => {
   const [valuesetMsg, setValuesetMsg] = useState(null);
 
   const getValueSetErrors = async (
-    valuesetsArray: ElmValueSet[],
-    tgtValue: string
+    valuesetsArray: ElmValueSet[]
   ): Promise<ElmTranslationError[]> => {
     const valuesetsErrorArray: ElmTranslationError[] = [];
-    if (valuesetsArray && tgtValue) {
+    if (valuesetsArray) {
```
Comment (jkotanchik-SB) [inline src/components/measureEditor/MeasureEditor.tsx:111]: I liked how the terminologyServiceApi wraps the Code validation call in a umlsLoggedIn check.

Either pass in the checkLogin value here similar to the call to validate Code, or check for it here and move the code validate check to this file as well.

File: src/components/measureEditor/MeasureEditor.tsx [hunk 2]
```diff
@@ -249,15 +250,10 @@ const MeasureEditor = () => {
...
 
       let valuesetsErrors = null;
-      if (
-        translationResults.library?.valueSets?.def !== null &&
-        tgt &&
-        tgtValue &&
-        tgtValue !== ""
-      ) {
+
+      if (translationResults.library?.valueSets?.def !== null && isLoggedIn) {
```
Comment (jkotanchik-SB) [inline src/components/measureEditor/MeasureEditor.tsx:254]: Why is the logged in check here and the Code validation check is in `useTerminologyServiceApi`?

Comment (sb-cecilialiu) [reply src/components/measureEditor/MeasureEditor.tsx:254]: @jkotanchik-SB Sorry just saw the comment, this is to validate value sets, only logged user can do it.
